charts/cinder-csi-plugin/values.yaml

```diff
@@ -70,10 +70,49 @@ secret:
 #      ca-file=/etc/cacert/ca-bundle.crt
 
 storageClass:
-  enabled: true
+  enabled: false
   delete:
     isDefault: false
     allowVolumeExpansion: true
   retain:
     isDefault: false
     allowVolumeExpansion: true
+# any kind of custom StorageClasses
+#   custom: |-
+#    ---
+#    apiVersion: storage.k8s.io/v1
+#    kind: StorageClass
+#    metadata:
+#      name: sas
+#    provisioner: cinder.csi.openstack.org
+#    volumeBindingMode: WaitForFirstConsumer
+#    allowVolumeExpansion: true
+#    parameters:
+#      type: SAS
+#    ---
+#    apiVersion: storage.k8s.io/v1
+#    kind: StorageClass
+#    metadata:
+#      name: ssd
+#    provisioner: cinder.csi.openstack.org
+#    volumeBindingMode: WaitForFirstConsumer
+#    allowVolumeExpansion: true
+#    parameters:
+#      type: SSD
+#    ---
+#    apiVersion: storage.k8s.io/v1
+#    kind: StorageClass
+#    metadata:
+#      name: sata
+#    provisioner: cinder.csi.openstack.org
+#    volumeBindingMode: WaitForFirstConsumer
+#    allowVolumeExpansion: true
+#    parameters:
+#      type: SATA
+#    apiVersion: snapshot.storage.k8s.io/v1beta1
```

Comment:
need to remove VolumeSnapshotClass def here?

Reply:
Seems what you need is to define `type` for the StorageClass, why not just add an option `volumeType`?

Reply:
sorry, the --- was in the wrong line